src/utils/isStakingSignReady.ts

```diff
@@ -39,6 +40,11 @@ export const isStakingSignReady = (
       isReady: false,
       reason: "Please enter a valid staking period",
     };
+  } else if (stakingFeeSat == 0) {
+    return {
+      isReady: false,
+      reason: "Not enough funds to cover fees for staking",
```

Comment:
Let's add a comment here as the `stakingFeeSat == 0` does not explain very well the situation behind the message.